public/app/plugins/datasource/alertmanager/types.ts

```diff
@@ -330,7 +330,7 @@ export interface TimeInterval {
 export type MuteTimeInterval = {
   name: string;
   time_intervals: TimeInterval[];
-  provenance?: string;
```

Comment:
Why the change? Isn't it the API response type?

Reply:
I _think_ this is the "UI" data model and not the DTO.